### ISSUE TITLE
Unification of name and description fields

### ISSUE THREAD

**Issue by mbredif** [LI3DS/pg_li3ds#2]
We have `transfo_type.func_name` and `sensor.short_name`. I propose to rename them as `transfo_type.name` and `sensor.name`.
More generally a proposal would be to generalize the fields to : 
```
id serial primary key -- used for foreign keys
name varchar unique not null -- textual name to be lookedup, more convenient for a human than the id
description varchar -- freeform text for documentation, to be consummed by humans only
```
for all types : project, platform, sensor, referential, session, transfo_type, transfo_tree, platform_config

**Comment by elemoine** [comment]
+1 from me!

**Comment by elemoine** [comment]
@mbredif do you have any explanation for this [FIXME](https://github.com/LI3DS/pg_li3ds/blob/master/extension/li3ds--1.0.0.sql#L47)?

**Comment by mbredif** [comment]
Nope. 
@ldgeo @ehabets ?

**Comment by elemoine** [comment]
I was about to replace it by `name` as discussed, but came across that FIXME…

**Comment by mbredif** [comment]
I think you can rename it and keep the comment. I think the fixme is just left out tmas an incomplete doc of how the name was chosen from other fields.

**Comment by ldgeo** [comment]
the FIXME doesn't make sense for me, you can remove it. +1 for the rest, we already discussed that.

**Comment by mbredif** [comment]
closed by #15
